pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -474,6 +474,43 @@ func (m *kubeGenericRuntimeManager) executePreStopHook(pod *v1.Pod, containerID
 	return int64(metav1.Now().Sub(start.Time).Seconds())
 }
 
+// executeRetryUntilSuccess runs the Retry-Until lifecycle hooks if applicable and returns the duration it takes.
+func (m *kubeGenericRuntimeManager) executeRetryUntilSuccessHook(pod *v1.Pod, containerID kubecontainer.ContainerID, containerSpec *v1.Container, gracePeriod int64) int64 {
+	glog.V(3).Infof("Running RetryUntilSuccess hook for container %q", containerID.String())
```

Comment:
nit: I think this should be V(4) (https://github.com/kubernetes/community/blob/master/contributors/devel/logging.md)